app.py

```diff
@@ -77,13 +70,14 @@ def create_sessions():
         return Response(json.dumps(status), status=400, mimetype='application/json')
 
 
-@app.route('/newapp', methods=['POST'])
+@app.route('/newapps', methods=['POST'])
 def handle_new_app():
     if request.method == 'POST':
         try:
-            app = request.get_json()['apps'][0]
+            verify_request(request.headers.get('App_Bot_Api_Secret'))
```

Comment:
The severless registry API puts the secret in the `Authorization` header, I can change that though.

Reply:
I'll put the secret wherever, if you could document where I should put it in [DESIGN.md](https://github.com/kscout/chat-bot-api/blob/master/DESIGN.md#new-app-upload) I'll update the serverless-registry-api accordingly

Reply:
Its better to put that in Authorization header. I will update the function and DESGIN.md. You don`t need to make any change

Reply:
Thank you!